src/compiler/typescript_to_rust/azle_generate/src/generators/vm_value_conversion/try_from_vm_value.rs

```diff
@@ -361,6 +368,12 @@ pub fn generate_try_from_vm_value_impls() -> proc_macro2::TokenStream {
             }
         }
 
+        impl CdkActTryFromVmValue<Vec<ic_cdk::timer::TimerId>, &mut boa_engine::Context> for boa_engine::JsValue {
```

Comment:
Just remember that this will need to be changed once we merge in the list of lists code